python/lsst/pipe/tasks/postprocess.py

```diff
@@ -790,6 +796,15 @@ def _makeArgumentParser(cls):
                                help="data ID, e.g. --id visit=12345 ccd=0")
         return parser
 
+    def runDataRef(self, dataRef):
+        """Override to specify band label to run()."""
+        parq = dataRef.get()
+        funcs = self.getFunctors()
+        band = dataRef.get("calexp_filterLabel", immediate=True).bandLabel
+        df = self.run(parq, funcs=funcs, dataId=dataRef.dataId, band=band)
```

Comment:
`TransformObjectCatalogTask.run` doesn't take a `band` keyword. Did you mean to put this in `TransformCatalogBaseTask`, or did you forget to add the keyword here?

Reply:
Ugh, and of course `TransformSourceTableTask` has no tests...

Wait, this is fine: it's calling the parent `TransformCatalogBaseTask.run()`, which does take `band`, and this is exactly why.

Reply:
Aha, I got confused by all the classes in the file. 😵 

However, it's still a violation of substitutability for `TransformObjectCatalogTask.run` to not take `band`, so please fix that.